java8-okhttp-gson/src/test/java/com/sphereon/sdk/pdf/api/Conversion2PDFApiTest.java

```diff
@@ -18,26 +18,33 @@
 import com.sphereon.sdk.pdf.model.ConversionJobResponse;
 import com.sphereon.sdk.pdf.model.ConversionSettings;
 import com.sphereon.sdk.pdf.model.Lifecycle;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
 import org.junit.Assert;
-import org.junit.FixMethodOrder;
-import org.junit.Test;
-import org.junit.runners.MethodSorters;
 
 import java.io.File;
+import java.io.IOException;
 import java.net.URL;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 
 /**
  * API tests for Conversion2PDFApi
  */
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
Why is this test not a test anymore? Unless I am missing something

Reply:
Ok got it. As you have overrides for every method that simply delegates to the super anyway. Why not simply leave as is and remove all the overrides? Just make this class abstract